tests/factory_fixtures/notes.py

```diff
@@ -11,11 +11,13 @@ def _note_attributes(text, user, ticket):
 
 
 @pytest.fixture
-def create_note(faker, note_attributes):
-    def _create_note(userid, ticketid, text=None):
-        if not text:
-            text = faker.paragraph()
-        note = NotesModel(**note_attributes(text, userid, ticketid))
+def create_note(faker, note_attributes, create_admin_user, create_ticket):
+    def _create_note(user=None, ticket=None, text=None):
+        text = text or faker.paragraph()
+        user = user or create_admin_user()
+        ticket = ticket or create_ticket()
+
+        note = NotesModel(**note_attributes(text, user, ticket))
         note.save_to_db()
```

Comment:
Since we're updating this file, let's change this to use the NoteModel's `.create` method so we can validate it with the schema.  Check out the `create_lease` fixture for an example of how to do this.

Reply:
Makes perfect sense. I made the change in a new commit.  We should probably hold off on merging it until #284 is pushed however.  The variable names "ticketid" and "userid" will need to be updated to bring it in line with the new terminology.  Thanks!